src/databricks/labs/remorph/intermediate/dag.py

```diff
@@ -6,8 +6,8 @@
 class Node:
     def __init__(self, name: str):
         self.name = name.lower()
-        self.children = []
-        self.parents = []
+        self.children: list[str] = []
+        self.parents: list[str] = []
 
     def add_parent(self, node: str):
```

Comment:
Missing return types for functions in entire file.

Reply:
Done